nodeadm/test/e2e/helpers.sh

```diff
@@ -102,9 +102,9 @@ function wait::dbus-ready() {
 
 function mock::aws() {
   local CONFIG_PATH=${1:-/etc/aemm-default-config.json}
-  imds-mock --config-file $CONFIG_PATH &
+  [ "${IMDS_MOCK_ONLY_CONFIGURE:-}" = "true" ] || imds-mock --config-file $CONFIG_PATH &
   export AWS_EC2_METADATA_SERVICE_ENDPOINT=http://localhost:1338
-  $HOME/.local/bin/moto_server -p5000 &
+  [ "${AWS_MOCK_ONLY_CONFIGURE:-}" = "true" ] || $HOME/.local/bin/moto_server -p5000 &
```

Comment:
maybe just split this into 2 helpers, `mock::imds` and `mock::aws`? I combined them so that our standard set of mocks didn't get unruly, but it's not that big of a deal

Reply:
i was considering it, but I would probably still leave an `ONLY_CONFIGURE` option so that i can enable the service part way through the run

Reply:
there's not much pressure since this is a pretty unique test so lets not worry about it for now :)